src/auth/index.js

```diff
@@ -208,19 +213,21 @@ export default class Auth {
     return this.storage.get(authCommon.REFRESH_TOKEN_KEY);
   }
 
-  set(json) {
-    if (!json) {
+  set(json, authType) {
+    if (!json || !authType) {
```

Comment:
It looks like there are cases where `authType` could definitely be empty which would cause us to not complete a log in. Is it necessary that we care what `authType` is?

Reply:
What is a case where `authType` could be empty? We don't care what `authType` is, but we care if it's `null`. It should be impossible to log in to stitch without a provider type.

Reply:
The handling of cookies and redirects